src/webui/layouts/home/Home.tsx

```diff
@@ -10,18 +10,18 @@ export type HomeProps = {
     aside: React.ReactNode;
 };
 
-export default function Home(props: HomeProps)
+export default function Home({ main, aside }: HomeProps)
```

Comment:
The name of the function should be `Layout` to be consistent with the rest of the pattern for webui implementations